src/native/managed/native-library.props

```diff
@@ -34,9 +33,4 @@
     <!-- passed by compile-native.proj to set - -gcc-toolchain=$(ROOTFS_DIR)/usr -->
     <CustomLinkerArg Condition="'$(CustomLinkerArgToolchainArg)' != ''" Include="$(CustomLinkerArgToolchainArg)" />
   </ItemGroup>
-
-  <ItemGroup>
```

Comment:
Conditionally include when MicrosoftDotNetILCompilerVersion is non-empty like in the other place?

Reply:
Lemme see if I can find a place to refactor this where it will get shared with evertyhing.

Reply:
OK, put everything in liveBuilds.targets. It's not the ideal location, but it's the best place I could find that's imported by everything that needs the override.